src/status_im/contexts/wallet/send/utils.cljs

```diff
@@ -43,21 +41,67 @@
   [route]
   (money/wei->ether (reduce money/add (map calculate-gas-fee route))))
 
+(defn- path-amount-in
+  [path]
+  (-> path :amount-in money/from-hex))
+
+(defn- path-amount-out
+  [path]
+  (if (-> path :bridge-name (= constants/bridge-name-hop))
+    (let [{:keys [token-fees bonder-fees amount-in]} path]
+      (-> amount-in
+          money/from-hex
+          (money/sub token-fees)
+          (money/add bonder-fees)))
+    (-> path :amount-out money/from-hex)))
```

Comment:
for the bridge out amount, we should show the value with the fees substracted, but not the bonder-fees, which are accounted for in the `estimated-received` value. This matches how [hop displays the bridge calculations](https://app.hop.exchange/#/send?sourceNetwork=ethereum&destNetwork=arbitrum&token=ETH).

<img width=250 src="https://private-user-images.githubusercontent.com/21865759/367665086-8ec19d5c-f339-4a18-a1f6-bf5add1af0d7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk2Njg1NDcsIm5iZiI6MTcyOTY2ODI0NywicGF0aCI6Ii8yMTg2NTc1OS8zNjc2NjUwODYtOGVjMTlkNWMtZjMzOS00YTE4LWExZjYtYmY1YWRkMWFmMGQ3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjMlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDIzVDA3MjQwN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUwN2Q3M2EwYWFkM2Q4MjQ4ZGFmNDFjODk5MDRkZDE4MWI5MjIyZTI1MzdmOGE3YTlhZTVlYWMwMTM4MDRhOGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.3fTaYGU7VrrEYV5pJRRBQipVqMYpamI69ylif0XE6tg" />